src/SPC/doctor/item/LinuxMuslCheck.php

```diff
@@ -26,7 +26,7 @@ public function checkMusl(): CheckResult
         }
 
         $musl_wrapper_lib = sprintf('/lib/ld-musl-%s.so.1', php_uname('m'));
-        if (file_exists($musl_wrapper_lib)) {
+        if (file_exists($musl_wrapper_lib) && file_exists('/usr/local/musl/bin/musl-gcc')) {
```

Comment:
I think this is the one file we shouldn't check, because there's really no point in enabling the gcc wrapper with our install. I will change this.

Reply:
Because I need to test the `/usr/local/musl` directory many times, I will delete this directory frequently and I must ensure that the directory exists.